test/checks/shared/avoid-inline-spacing.js

```diff
@@ -114,8 +114,19 @@ describe('avoid-inline-spacing tests', function() {
 		var vNode = queryFixture(
 			'<p id="target" style="line-height: 3 !important; letter-spacing: 50rem !important;">The quick brown fox jumped over the lazy dog</p>'
 		);
-		var actual = check.evaluate.call(checkContext, vNode.actualNode);
+		var actual = checkEvaluate.call(checkContext, vNode.actualNode);
 		assert.isFalse(actual);
 		assert.deepEqual(checkContext._data, ['line-height', 'letter-spacing']);
 	});
+
+	it('supports options.cssProperties', function() {
+		var vNode = queryFixture(
+			'<p id="target" style="font-size: 14px !important;">The quick brown fox jumped over the lazy dog</p>'
```

Comment:
Just to also show it doesn't flag the other props:

```suggestion
			'<p id="target" style="font-size: 14px !important; line-height: 3 !important; letter-spacing: 50rem !important">The quick brown fox jumped over the lazy dog</p>'
```